envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/rbac/RBACFilterFactoryJwtTest.kt

```diff
@@ -257,7 +305,9 @@ internal class RBACFilterFactoryJwtTest : RBACFilterFactoryTestUtils {
     private fun getTokenFieldForClientWithSelector(provider: String, client: String) =
         jwtProperties.providers[provider]!!.matchings[client]!!
 
-    private fun oAuthClientPrincipal(selectorMatching: String, selector: String) = """{
+    private fun oAuthClientPrincipal(selectorMatching: String, selector: String, negated: Boolean = false) =
```

Comment:
Maybe split to separate functions? Or even just add `"invert": true` conditionally

Reply:
changed